google/ggrc-core
# GGRC-3788 Fix js error which occurs when expanding tree view

# Issue description

"Uncaught TypeError: Cannot read property 'related' of undefined" error occurs while expanding tree view

# Steps to test the changes

1. Go to My Work page > Assessments tab
2. Hover over gear icon on the first tier and click Expand tree
3. Look at the screen: JS error is displayed 

**Actual Result:** GGRC-DEV: "Uncaught TypeError: Cannot read property 'related' of undefined" error occurs while expanding tree view
**Expected Result:** no error should be displayed while expanding subtree for all objects
**NOTE:** an error occurs for all entities

# Solution description

Regression issue - #6728 PR. Issue happens after migrating tree view utils on module syntax.
Migrate current page utils on module syntax as there is a cyclic dependency between tree view and current page utils.

# Sanity checklist




# PR Review checklist
